packages/patternfly-4/react-table/src/components/Table/demo/CustomRowWrapper.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+
+export default class CustomRowWrapper extends React.Component {
+  render() {
+    const {
+      trRef,
+      className,
+      rowProps,
+      row: { isExpanded, isHeightAuto },
+      ...props
+    } = this.props;
+
+    const isEvenOddRow = (rowProps.rowIndex + 1) % 2;
```

Comment:
Do you think this prop could possibly be renamed to `isOddRow`? I was confused by the name of it since in this example you're applying the styles to only the odd rows